test/unit/ngsiv2/examples/contextAvailabilityRequests/updateIoTAgent1.json

```diff
@@ -3,7 +3,7 @@
         "entities": [
             {
                 "type": "Light",
-                "id": "light1"
+                "id": "Light:light1"
```

Comment:
Does the change in this existing test mean a change of existing behavior?

Reply:
As per the existing behaviour entity_name can't be changed, so this id was not into effect. Now, if the entity_name is allowed to get updated then it should be same as per the standards. It will not change the existing behaviour.

Reply:
NTC